src/react/workflow/ReplicationForm.tsx

```diff
@@ -286,14 +286,34 @@ function ReplicationForm({ prefix = '', isCreateMode, ...props }: Props) {
                         readonly={isEditing}
                         disableOption={({
                           isBucketVersionned,
-                          locationInfos,
+                          isReplicationSourceSupported,
                           isVeeamBucket,
-                        }) =>
-                          !locationInfos?.storageOption
-                            ?.supportsReplicationSource ||
-                          !isBucketVersionned ||
-                          isVeeamBucket
-                        }
+                        }) => ({
+                          disabled:
+                            isReplicationSourceSupported ||
+                            !isBucketVersionned ||
+                            isVeeamBucket,
+                          disabledMessage: isReplicationSourceSupported ? (
```

Comment:
Can we extend the existing workflow tests to check the presence of the tooltip ?